src/test/java/uk/gov/justice/digital/hmpps/vsip/services/PrisonVisitsTestingHelperService.java

```diff
@@ -76,14 +78,35 @@ public void startPrisonerRestriction(String prisonerCode) {
         client.put("/test/prisoner/restriction", request, client.validateCreateStatusHandler, "Prisoner restriction not created");
     }
 
-    public void cleanUpBookings() {
-        LOG.debug("Entered cleanUpLastBooking");
-        var bookings = context.getBookingReferences();
-        if (bookings != null) {
-            LOG.debug(bookings.size() + " Bookings found, must tidy up!");
-            // We use changing here because it is not counted as part of booking slot counts and is also removed after configured time.
-            bookings.forEach(bookingReference -> client.changeStatus(bookingReference, VisitStatus.CHANGING));
+    public void cleanUpBookingsAndApplications() {
+        LOG.debug("Entered cleanUpLastBooking"
+        );
+        if (context.getBookingReferences()!=null) {
+            var bookings = new ArrayList<>(context.getBookingReferences());
+            if (bookings != null) {
+                LOG.debug(bookings.size() + " Bookings found, must tidy up!");
+                // We use changing here because it is not counted as part of booking slot counts and is also removed after configured time.
```

Comment:
Don't think this comment is still accurate as `CHANGING` status no longer relevant?